src/pages/network/connections/connections.service.ts

```diff
@@ -0,0 +1,67 @@
+import { get } from 'src/core/http';
+import { toRelativeTime } from 'src/core/relative-time';
+import { ConnectStatus, ConnectionItem } from 'src/core/types';
+
+export function getConnections(payload: {
+  page: number;
+  status?: ConnectStatus;
+  requester_id?: string;
+  requested_id?: string;
+}) {
+  return get(
```

Comment:
Any reason you directly called `get` instead of using endpoints in `endpoints/index`? Did I write this code?

Reply:
I don't know about your logics you used somewhere you used directly and somewhere you used `endpoint`, i think you used directly when you called in loading of page.
If it is not so, i will change it.

Reply:
We should never use `get` directly unless there is a good reason. Ehsan might have used it. If I have used it somewhere it's because we did not have `endpoint.ts` at that time.